dxapp.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "dnanexus_nirvana_2.0.3",
+  "title": "dnanexus_nirvana_2.0.3",
+  "summary": "dnanexus_nirvana_2.0.3",
+  "tags:": ["Annotation"],
+  "properties": {
+    "github release": "v1.0.0"
+  },
+  "dxapi": "1.0.0",
+  "inputSpec": [
+    {
+      "name": "input_vcf",
+      "label": "Input VCF file",
+      "help": "A VCF file containing variants to be annotated",
+      "class": "file",
+      "optional": false,
+      "patterns": ["*.refseq_nirvana_203.vcf"]
+    }
+  ],
+  "outputSpec": [
+    {
+      "name": "output_json",
+      "label": "Output JSON.gz file",
+      "help": "A JSON.gz file containing a structured JSON representation of all annotation and sample information",
+      "class": "file",
+      "patterns": ["*.json.gz"]
+    }
+  ],
+  "runSpec": {
+    "timeoutPolicy": {
+      "*": {
+        "hours": 48
+      }
+    },
+    "interpreter": "bash",
+    "release": "16.04",
+    "distribution": "Ubuntu",
+    "file": "src/dnanexus_nirvana_2.0.3.sh"
+  },
+    "access": {
+    "project": "CONTRIBUTE",
+    "allProjects": "VIEW",
+    "network": [
+      "*"
+    ]
+  },
+  "regionalOptions": {
+    "aws:us-east-1": {
+      "systemRequirements": {
+        "*": {
+          "instanceType": "mem1_ssd1_v2_x4"
```

Comment:
Did you try other configurations?

Reply:
This is the smallest workable instance, fewer cores gives insufficient disk space. Most of the run time is taken up by downloading and unpacking the data (~13 of 15 mins). Once everything is set up the annotation only takes ~2 mins for a clinical exome so I don't expect that we would see much benefit from going higher spec.

Reply:
:+1:

Reply:
I forgot you can't see jobs costs yet